docs/devops/explorer.md

```diff
@@ -7,5 +7,46 @@ sidebar_position: 4
 import useBaseUrl from "@docusaurus/useBaseUrl";
 
 # Explorer
+Axon uses [BlockScan](https://github.com/Magickbase/blockscan), an explorer tailored for Ethereum-compatible chains. By deploying BlockScan, you can access data from Axon-based chains and monitor their status.
 
-🛠 This page is a work in progress. 🚧
+## Deployment
+
+Docker Compose is the current deployment method.
+
+### Docker Compose
+
+1. Clone BlockScan repository to your target machine:
+
+``` shell
+git clone https://github.com/Magickbase/blockscan.git
+cd blockscan
+```
+
+2. Edit the config file located at `- blockscan/dev.env`. Make the following adjustments:
```

Comment:
`dev.env` is not mounted in docker-compose file by default.
@Simon-Tl

Reply:
The reason I want to mount by default is
For developers to connect other axon nodes to explorer, more convenient modification

Reply:
I think the explanation is needed.

Or,
As a new developer to Axon, I may assume that `dev.env` will be used in `docker-compose` after reading the docs.

So, I think it's better to mount dev.env by default, using cmd similar to:

```bash
docker-compose --env-file ./dev.env ...
```

Reply:
Yes, I've made dev.env mount by default inside docker-compose, and if dev.env doesn't exist, docker-compose won't work